src/com/company/model/CreateTableQueryBuilder.java

```diff
@@ -0,0 +1,38 @@
+package com.company.model;
+
+import java.sql.SQLException;
+
+import java.util.List;
+
+/**
+ * Created by yulia on 21.02.17.
+ */
+public class CreateTableQueryBuilder {
+
+    DatabaseManager databaseManager;
+
+    public boolean queryBuildExecute(Integer columnNumber, String tableName) throws SQLException {
```

Comment:
This method does at least two things:
1. Build SQL query.
2. Execute this query.
Good candidate for refactoring by splitting into 2 separate methods: **build**(...) and **execute**(...).